<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlMSSearchOption</class>
 <widget class="QWidget" name="ControlMSSearchOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MS Search Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="framePolarity">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Detection mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioPositiveIon">
       <property name="text">
        <string>Positi&amp;ve ionization</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioNegativeIon">
       <property name="text">
        <string>&amp;Negative ionization</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frameMassShift">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Mass tolerance:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textMassTolerance">
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioAbsoluteTolerance">
        <property name="text">
         <string>Absol&amp;ute (Da)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioRelativeTolerance">
        <property name="text">
         <string>&amp;Relative (ppm)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="frameCommand">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
